project/Build.scala

```diff
@@ -0,0 +1,49 @@
+import com.typesafe.sbt.{GitPlugin, SbtScalariform}
+import de.heikoseeberger.sbtheader.HeaderPlugin
+import de.heikoseeberger.sbtheader.license.Apache2_0
+import sbt._
+import sbt.plugins.JvmPlugin
+import sbt.Keys._
+import sbtassembly.AssemblyPlugin
+
+import scalariform.formatter.preferences.{AlignSingleLineCaseStatements, DoubleIndentClassDeclaration}
+
+object Build extends AutoPlugin {
+
+  override def requires = JvmPlugin && HeaderPlugin && GitPlugin && SbtScalariform
+
+  override def trigger = allRequirements
+
+  override def projectSettings = Vector(
+    // Core settings
+    organization := "de.bripkens",
+    licenses += ("Apache-2.0", url("http://www.apache.org/licenses/LICENSE-2.0")),
```

Comment:
The project is currently using the MIT license, but we can switch to Apache 2.0. The only thing we need to account for is consistency. Meaning: We need to change the license reference in the README and the LICENSE file.